### ISSUE TITLE
[Feature Request] Date picker (range) closing only when the range selection is finished

### ISSUE THREAD

**Issue by fadafez** [vuetifyjs/vuetify#9984]
### Problem to solve
When I create a date picker with range mode activated, I've tried the suggested examples in the docs, but the input event gets fired when each date is clicked, and not when the user selects the final date.

### Proposed solution
Actually I've added a watcher method on the property linked to my datepicker (let's call it `daterange`). 

```
 watch: {
      period: function (newValue, oldValue) {
        if (newValue !== oldValue && this.daterange.length === 2) {
          this.getRankings() // i need to update some tables 
        }
      }
    },
```

Another solution (not tested!) could be to make the `@input="rangeDialog = updateTables()"`

```
 updateTables () {
        if (this.period.length === 2) {
          this.getRankings()
          return false
        }
      },
```

So when the array is full, I now that the user selected start and end dates, so I can return false to rangeDialog, and it will close the datepicker dialog / menu. 

I needed this solution because there's an API call in the middle, so anytime the dates are changed, I want a table to update accordingly, without the user clicking any buttons. 

I don't know if it's elegant but it works. 

I was also looking for a reference to `:return-value.sync `property for `v-dialog`, but I cannot find it in the documentation, maybe I've missed it in another generic section, but the search engine unfortunately is not returning a link to it. 

Cheers!

**Comment by jacekkarczmarczyk** [comment]
With https://github.com/vuetifyjs/vuetify/pull/9761 you can choose whether you want to do something after any date changes or after full range is selected
Also this kind of issues need a reproduction so we know what code you're about

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by ShawshankLin** [comment]
why not design like this？ https://codepen.io/pen/?&editable=true

**Comment by jacekkarczmarczyk** [comment]
Not a big fan of this solution
![image](https://user-images.githubusercontent.com/15625235/71079347-b0627700-21bd-11ea-9c7d-6fb929348b57.png)

**Comment by ShawshankLin** [comment]
> Not a big fan of this solution
> ![image](https://user-images.githubusercontent.com/15625235/71079347-b0627700-21bd-11ea-9c7d-6fb929348b57.png)

sorry, the correct address is  https://element.eleme.cn/#/zh-CN/component/date-picker#xuan-ze-ri-qi-fan-wei, the date panel  can be linked

**Comment by fadafez** [comment]
Hi @ShawshankLin, when I opened the first link I tought it was a joke... LOL :-) 
Anyway, I was using ElementUI before of vuetify, the date range picker was the main reason of why I choosed that library instead of the others. 
Then I had to switch to vuetify because of the lack of support for responsiveness, which my client asked later on. 

I'm not a huge fan of mixing two libraries, so I had to do a complete switch.

@jacekkarczmarczyk Thanks for the information, I will adapt the code with the new feature. I've opened a bug report yesterday (#10162), there I've attached a codepen to simulate the problem.

See you the next time, and thanks for the wonderful library
